src/util/postgres_client.js

```diff
@@ -1331,7 +1355,7 @@ class PostgresClient extends EventEmitter {
         this.new_pool_params = {
 
             // TODO: check the effect of max clients. default is 10
-            // max: 50,
+            max: (process.env.LOCAL_MD_SERVER === 'true') ? 80 : 10,
```

Comment:
I suggest adding 'POSTGRES_MAX_CLIENTS` to config.js, which gives us the ability to override it with an env.